pkg/sync/v6/sync.go

```diff
@@ -3,6 +3,8 @@ package sync
 //go:generate go get github.com/go-bindata/go-bindata/go-bindata
 //go:generate go-bindata -nometadata -pkg $GOPACKAGE -prefix data data/...
 //go:generate gofmt -s -l -w bindata.go
+//go:generate curl https://raw.githubusercontent.com/openshift/cluster-monitoring-operator/master/assets/cluster-monitoring-operator/cluster-role.yaml -o data/ClusterRole.rbac.authorization.k8s.io/geneva-cluster-monitoring-view.yaml
+//go:generate sed -i "s/cluster-monitoring-view/geneva-cluster-monitoring-view/" data/ClusterRole.rbac.authorization.k8s.io/geneva-cluster-monitoring-view.yaml
```

Comment:
i'd prefer this in the content upgrade generator, i'll see how to get it in there... @ehashman just wanted to make this automatic so it doesn't deviate from upstream..